src/collections/_documentation/platforms/php/index.md

```diff
@@ -345,16 +356,50 @@ The best adapter available is automatically selected when creating a client inst
 through the client builder, but you can override it using the appropriate methods.
 
 ```php
+use Http\Discovery\HttpAsyncClientDiscovery;
+use Http\Discovery\MessageFactoryDiscovery;
+use Http\Discovery\StreamFactoryDiscovery;
+use Http\Discovery\UriFactoryDiscovery;
+use Http\Message\ResponseFactory;
+use Http\Message\UriFactory;
 use Sentry\ClientBuilder;
+use Sentry\HttpClient\HttpClientFactory;
+use Sentry\HttpClient\HttpClientFactoryInterface;
 use Sentry\Transport\HttpTransport;
 use Sentry\State\Hub;
+use Sentry\Transport\TransportFactoryInterface;
 
 $options = ['dsn' => '___PUBLIC_DSN___'];
```

Comment:
I would just inline this variable since we won't reuse it besides in the `ClientBuilder::create` method